content/en/blog/argo-insurance-workflow/index.md

```diff
@@ -0,0 +1,126 @@
+
+### Argo Workflows
+
+For installing the latest version Argo Workflows, you can refer to [Argo Workflows Installation](https://argoproj.github.io/argo-workflows/quick-start/)  
+
+We are ready to build our Insurance Installment calculator now.
+
+### Creating Fission functions and triggers
+
+We have to create all the functions which would be helping us with the calculation. You can refer to [Fission Function](https://github.com/neha-Gupta1/argo-workflow/tree/main/insurance/fission-functions) folder for all the functions required for this application. Clone the above folder and run below command -
+
+`sudo chmod 755 insurance/fission-functions/spec.sh`
+
+Verifying the Fission function and Trigger creation,
+
+```bash
+$ fission fn list
+NAME              ENV    EXECUTORTYPE MINSCALE MAXSCALE MINCPU MAXCPU MINMEMORY MAXMEMORY SECRETS CONFIGMAPS
+get-full-name     go     poolmgr      0        0        0      0      0         0
+highriskinsurance go     poolmgr      0        0        0      0      0         0
+lowriskinsurance  go     poolmgr      0        0        0      0      0         0
+set-age           go     poolmgr      0        0        0      0      0         0
+set-first-name    go     poolmgr      0        0        0      0      0         0
+set-last-name     go     poolmgr      0        0        0      0      0         0
+set-salary        go     poolmgr      0        0        0      0      0         0
+```
+
+```bash
+$ fission httptrigger list
+NAME                                 METHOD URL                     FUNCTION(s)       INGRESS HOST PATH                    TLS ANNOTATIONS
+65222407-76ed-43b2-97d9-160d5bc61c36 [GET]  /hello                  hello             false   *    /hello                      
+getfullname                          [GET]  /getfullnamehandler     get-full-name     false   *    /getfullnamehandler         
+gethighriskinsurance                 [GET]  /getHighRiskEligibility highriskinsurance false   *    /getHighRiskEligibility     
+getlowriskinsurance                  [GET]  /getLowRiskEligibility  lowriskinsurance  false   *    /getLowRiskEligibility      
+postage                              [GET]  /setage                 set-age           false   *    /setage                     
+postfirstname                        [GET]  /postfirstname          set-first-name    false   *    /postfirstname              
+postlastname                         [GET]  /postlastname           set-last-name     false   *    /postlastname               
+postsalary                           [GET]  /setmonthlysalary       set-salary        false   *    /setmonthlysalary  
+```
+
+### Running the application
+
+The above function helped us in creating independent logics. Now we are going to club them together and create a workflow using Argo Workflows. This workflow takes its decision on basis of input provided.
```

Comment:
- We used  shell script to generate all functions and triggers instead of functions, let's rephrase that.
- this workflow takes its decision on `the` basis of `inputs` provided - _add the and s input_